.yamato/yamato-ci.yml

```diff
@@ -0,0 +1,67 @@
+linux:
+  name: "Linux"
+  agent:
+    type: Unity::VM
+    image: cds-ops/cds-ubuntu-18.04-base:stable
+    flavor: b1.small
+  commands:
+    - git submodule init
+    - git submodule update
+    - mono bee.exe build::tundra2::master
+    - mono bee.exe run::tundra2-unittest::master
```

Comment:
Still not in love with using Bee to launch tests. I'd be a little happier if it was more like "build the test-results report file"...